proxy/ui.coffee

```diff
@@ -69,6 +69,18 @@ class WebExtUI extends UI
       @postActive()
     ), 60 * 60 * 1000
 
+  initToggle: ->
+    try
+      getting = chrome.storage.local.get("snowflake-enabled", (result) =>
+        if result['snowflake-enabled'] != undefined
+          @enabled = result['snowflake-enabled']
+          @setEnabled @enabled
+        else
+          log "Toggle state not yet saved"
```

Comment:
Need to call `@setEnabled` unconditionally here now that it's run in `init`, otherwise if there's no saved state, it doesn't default to enabling.

Also, why the `try/catch`?  Do we expect this to fail?

Reply:
Thanks, made this change.

The try catch was there because I thought I'd needed it originally but it turned out I was using storage.local.get wrong and forgot to change it back when I corrected this. I've removed it now.